test/unit/iotagent-thinking-things_test.js

```diff
@@ -173,6 +185,40 @@ describe('Southbound measure reporting', function() {
         it('should return a 200OK with the configured sleep time in the core module',
             checkResponse(options, '#STACK1#5143,GPS,-1$cond1,#673495,K1,300$theCondition,'));
     });
+
+
+
+    describe('When a request arrives to the IoT Agent having Generic Configuration modules', function() {
+        var options = {
+            url: 'http://localhost:' + config.thinkingThings.port + config.thinkingThings.root + '/Receive',
+            method: 'POST',
+            form: {
+                cadena: '#STACK1#0,GC,conf,33,600$,#1,GC,conf2,123,600$,#673495,K1,2500$theCondition,'
```

Comment:
We clarified online that `cadena` is part of the Thinking Things protocol so it is perfectly fine ;)